test/openshift/e2e/sequential/1-077_validate_workload_status_monitoring_alert/01-install.yaml

```diff
@@ -1,12 +1,12 @@
-# apiVersion: argoproj.io/v1alpha1
-# kind: ArgoCD
-# metadata:
-#   name: openshift-gitops
-#   namespace: openshift-gitops
-#   labels:
-#     example: argocd
-# spec:
-#   applicationSet: 
-#     image: test-image
-#   monitoring:
-#     enabled: true
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: openshift-gitops
+  namespace: openshift-gitops
```

Comment:
As @rjeczkow mentioned the test leaves applicationset-controller in ImagePullBackOff (to check if the alert is firing). We should probably not use openshift-gitops namespace for it?

Reply:
If using `openshift-gitops` namespace is not necessary (The alerts would fire for any other argo-cd instance I believe) we could use argo-cd in other namespace and place this test in 'parallel' suite because it wouldn't modify the default argo-cd instance any more. Then we could keep the `sleep 10` since it wouldn't block other tests execution.